src/server/scripts/Commands/cs_guild.cpp

```diff
@@ -180,6 +181,65 @@ class guild_commandscript : public CommandScript
         uint8 newRank = uint8(atoi(rankStr));
         return targetGuild->ChangeMemberRank(targetGuid, newRank);
     }
+
+    static bool HandleGuildInfoCommand(ChatHandler* handler, char const* args)
+    {
+        Player* target;
+        uint32 guildId;
+        std::string guildName;
+        std::string guildMasterName;
+        Guild* guild;
+
+        if (!*args)
+        {
+            // Look for the guild of the selected player or ourselves
+            if (target = handler->getSelectedPlayerOrSelf())
+                guild = target->GetGuild();
+            else
+                // getSelectedPlayerOrSelf will return null if there is no session
+                // so target becomes nullptr if the command is ran through console
+                // without specifying args.
+                return false;
+        }
+        else if (guildId = atoi(args)) // Try searching by Id
+            guild = sGuildMgr->GetGuildById(guildId);
+        else
+        {
+            // Try to extract a guild name
+            char* tailStr = *args != '"' ? strtok(nullptr, "") : (char*)args;
+            if (!tailStr)
+                return false;
+
+            char* guildStr = handler->extractQuotedArg((char*)args);
+            if (!guildStr)
+                return false;
+
+            guildName = guildStr;
+            guild = sGuildMgr->GetGuildByName(guildName);
+        }
+
+        if (!guild)
+            return false;
+
+        // Display Guild Information
+        handler->PSendSysMessage(LANG_GUILD_INFO_NAME, guild->GetName().c_str(), guild->GetId()); // Guild Id + Name
+        if (sObjectMgr->GetPlayerNameByGUID(guild->GetLeaderGUID(), guildMasterName))
+            handler->PSendSysMessage(LANG_GUILD_INFO_GUILD_MASTER, guildMasterName.c_str(), guild->GetLeaderGUID()); // Guild Master
```

Comment:
Guid is uint64 and the format in DB is `| Guild Master: %s (GUID: %u)`. This means that you should instead print lowguid in the message. That is uint32.